wandb/wandb
# security(launch): Warn when agent is started on a team queue

Fixes [WB-12826](https://wandb.atlassian.net/browse/WB-12826)

Description
-----------
When the agent is started, print a warning if the assigned entity is a team 

Testing
-------
- `pip install -e .`
- Start launch agent with ` wandb launch-agent -q kubernetes-queue -e launch-test`
- Confirm the warning message appears (see screenshot below)
- Also confirmed the warning message doesn't appear on a personal entity
- A unit test was also added to test this behavior

![image](https://user-images.githubusercontent.com/39655500/224167735-bd4f696b-ff57-49c9-9a27-60e8dd44c40d.png)



[WB-12826]: https://wandb.atlassian.net/browse/WB-12826?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
